pkg/arvo/sys/vane/ames.hoon

```diff
@@ -965,6 +1057,28 @@
         message-pump-state
       ship-state
     ames-state
+  ::
+  ++  state-5-to-6
+    |=  ames-state=ames-state-5
+    ^-  ^^ames-state
+    :_  +.ames-state
+    %-  ~(rut by peers.ames-state)
+    |=  [=ship ship-state=ship-state-5]
+    ^-  ^ship-state
+    ?.  ?=(%known -.ship-state)
+      ship-state
+    =/  peer-state=peer-state-5  +.ship-state
+    =|  =rift
+    =/  scry=(unit (unit cage))
+      (rof ~ %j `beam`[[our %rift %da now] /(scot %p ship)])
+    =?  rift  ?=([~ ~ ^] scry)
```

Comment:
I'm failing to see how this would crash... Looking in `%jael` the scry gives us the rift if it succeeds, `~` if it can't find the ship, or `[~ ~]` if we called it with a wrong beam.